src/main/java/picard/vcf/CreateEvenIntervals.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2012 The Broad Institute
```

Comment:
![](https://img.memesuper.com/3e873811593d9f0b19b30c0e39a59fb5_-may-jumanji-what-year-is-it-meme_666-570.jpeg)